pdal/util/portable_endian.hpp

```diff
@@ -54,6 +54,14 @@
 #   include <winsock2.h>
                      
 #   if BYTE_ORDER == LITTLE_ENDIAN
+
```

Comment:
Where did the implementation for htonll and ntohll come from? Please add a comment.

Reply:
i think from so:
https://stackoverflow.com/questions/16375340/c-htonll-and-back